docs/reference/eager.md

```diff
@@ -0,0 +1,561 @@
+## EAGER Toolkit
+
+Engine Agnostic Gym Environment for Robotics (EAGER) is a toolkit that
```

Comment:
Could you please update the documentation so that each sentence is written on a single line and each new sentence in a new line.

This is the common style used for documentation and it makes easier to review it.